### ISSUE TITLE
Current implementation needs transpiled circuit

### ISSUE THREAD

**Issue by jf-kong** [qiboteam/qibo-cloud-backends#4]
Following on the discussion during the weekly meeting, I think the current implementation needs the circuit to be transpiled by the user before execution, as the `backend.run()` method does not do transpilation. Perhaps this can be made clear in the documentation.

**Comment by scarrazza** [comment]
@BrunoLiegiBastonLiegi could you please double check @jf-kong comment?

**Comment by BrunoLiegiBastonLiegi** [comment]
Yes, we don't perform any transpilation here and the qiskit backend doesn't perform transpilation as well. There is already a note in the documentation about this.
